charts/bcbox/values.yaml

```diff
@@ -115,6 +115,7 @@ config:
     FRONTEND_COMS_APIPATH: ~
     FRONTEND_OIDC_AUTHORITY: ~
     FRONTEND_OIDC_CLIENTID: ~
+    FRONTEND_NOTIFICATION_HOME_BANNER: ~
```

Comment:
Please have this be inserted in alphabetical order. In this case, it should come before the OIDC keys.
Also note that since we are doing a helm chart change, we need to increment the patch version of this chart in Charts.yaml for this to take any effect after it gets merged.

Reply:
Fixed, bumped chart version